src/appshell/internal/applicationactions.cpp

```diff
@@ -28,6 +28,10 @@ const ActionList ApplicationActions::m_actions = {
     ActionItem("quit",
                ShortcutContext::Any,
                QT_TRANSLATE_NOOP("action", "Quit")
+               ),
+    ActionItem("fullscreen",
+               ShortcutContext::Any,
+               QT_TRANSLATE_NOOP("action", "Full Screen")
```

Comment:
Is there any description for this action?

Reply:
thanks, added description